Shared/Objects/VideoPlayerType.swift

```diff
@@ -22,4 +22,381 @@ enum VideoPlayerType: String, CaseIterable, Defaults.Serializable, Displayable {
+                    container: MediaContainer.mpegts.rawValue,
+                    type: .video,
+                    videoCodec: [
+                        VideoCodec.h264,
+                    ].map(\.rawValue).joined(separator: ",")
+                ),
+
+                DirectPlayProfile(
+                    audioCodec: [
+                        AudioCodec.aac,
+                        AudioCodec.amr_nb,
+                    ].map(\.rawValue).joined(separator: ","),
+                    container: [
+                        MediaContainer.threeGP,
+                        MediaContainer.threeG2,
+                    ].map(\.rawValue).joined(separator: ","),
+                    type: .video,
+                    videoCodec: [
+                        VideoCodec.h264,
+                        VideoCodec.mpeg4,
+                    ].map(\.rawValue).joined(separator: ",")
+                ),
+
+                DirectPlayProfile(
+                    audioCodec: [
+                        AudioCodec.pcm_s16le,
+                        AudioCodec.pcm_mulaw,
+                    ].map(\.rawValue).joined(separator: ","),
+                    container: MediaContainer.avi.rawValue,
+                    type: .video,
+                    videoCodec: [
+                        VideoCodec.mjpeg,
+                    ].map(\.rawValue).joined(separator: ",")
+                ),
+            ]
+        case .swiftfin:
+            return [
+                DirectPlayProfile(
+                    audioCodec: [
+                        AudioCodec.aac,
+                        AudioCodec.ac3,
+                        AudioCodec.alac,
+                        AudioCodec.amr_nb,
+                        AudioCodec.amr_wb,
+                        AudioCodec.dts,
+                        AudioCodec.eac3,
+                        AudioCodec.flac,
+                        AudioCodec.mp1,
+                        AudioCodec.mp2,
+                        AudioCodec.mp3,
+                        AudioCodec.nellymoser,
+                        AudioCodec.opus,
+                        AudioCodec.pcm_alaw,
+                        AudioCodec.pcm_bluray,
+                        AudioCodec.pcm_dvd,
+                        AudioCodec.pcm_mulaw,
+                        AudioCodec.pcm_s16be,
+                        AudioCodec.pcm_s16le,
+                        AudioCodec.pcm_s24be,
+                        AudioCodec.pcm_s24le,
+                        AudioCodec.pcm_u8,
+                        AudioCodec.speex,
+                        AudioCodec.vorbis,
+                        AudioCodec.wavpack,
+                        AudioCodec.wmalossless,
+                        AudioCodec.wmapro,
+                        AudioCodec.wmav1,
+                        AudioCodec.wmav2,
+                    ].map(\.rawValue).joined(separator: ","),
+                    type: .video,
+                    videoCodec: [
```

Comment:
As far as I can tell, this is a change from before for supported Swiftfin (VLCKit) video codecs. Before, none were defined, meaning all were permitted. As I mentioned previously this was based on what I now realize was a bad assumption. Regardless, I believe the list should include the following additional not listed here: vp8, msmpeg4v3, msmpeg4v2, msmpeg4v1, theora, ffv1, flv1, wmv3, wmv2, wmv1. If you see an error with those, please let me know!

Reply:
Shoot yes it is. I've done too much copying and pasting on revisions. Let me make sure those are all back where they should be.

Standby!

Reply:
Apologies on that, that part of the profile was copied from the Transcode Profile. I've remove the video restrictions on VLC. Additionally, I re-added your comments in this VideoPlayerType layout so we don't lose that knowledge with this reorganization!

Please let me know if you see anything else like this!